runtime-builder/build.sh

```diff
@@ -26,12 +26,12 @@ usage() { echo "Usage: $0 <project> <go_version>"; exit 1; }
 base_digest()
 {
   # We sometimes have issue with 'describe', use the --log-http flag to get more info if it fails.
```

Comment:
I will change the comment as a note just to remind us that it uses different permission than other images command.